WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -496,7 +496,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // When swiping between different sections, select the corresponding
         // tab. We can also use ActionBar.Tab#select() to do this if we have
         // a reference to the Tab.
-        mViewPager.setOnPageChangeListener(new ViewPager.SimpleOnPageChangeListener() {
+        mViewPager.addOnPageChangeListener(new ViewPager.SimpleOnPageChangeListener() {
```

Comment:
Similar to my earlier comment about for `addOnTabSelectedListener`, we may want to clear the existing listeners before this call with `mViewPager.clearOnPageChangeListeners();` just to be on the defensive side.

Again, it's your call for whether to include that in this PR or not.

Reply:
Added